src/ussd_pmtct.js

```diff
@@ -0,0 +1,254 @@
+go.app = function() {
+    var vumigo = require("vumigo_v02");
+    var App = vumigo.App;
+    var Choice = vumigo.states.Choice;
+    var ChoiceState = vumigo.states.ChoiceState;
+    var EndState = vumigo.states.EndState;
+    var FreeText = vumigo.states.FreeText;
+
+
+    var GoNDOH = App.extend(function(self) {
+        App.call(self, "state_start");
+        var $ = self.$;
+        // var interrupt = true;
+
+        self.init = function() {
+            // init services
+        };
+
+    // TEXT CONTENT
+        var questions = {
```

Comment:
@NicolaasDieBaas Do you feel like this setup of having the question text up here and not in the states adds value? A few times with our other apps like this I've thought it's not worth it.